data/src/main/java/org/oppia/data/backends/gae/ModifyJsonInterceptor.kt

```diff
@@ -0,0 +1,30 @@
+package org.oppia.data.backends.gae
+
+import okhttp3.*
+
+import java.io.IOException
+
+class ModifyJsonInterceptor : Interceptor {
```

Comment:
For here & all other new classes in this PR: please add Javadoc documentation for them.

Reply:
Added Javadoc to every class and also changed ModifyJsonInterceptor to NetworkInterceptor